opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -668,9 +680,24 @@ namespace cpgrid
 
             // Build face to cell.
             f2c.clear();
-            int nf = output.number_of_faces;
             cpgrid::EntityRep<0> cells[2];
             face_to_output_face.clear();
+            // Add nnc connections first, to preserve the
+            // property that top and bottom faces come last
+            // (per cell) in the cell-face mapping.
+            for (const auto& nncpair : nnc) {
+                const int c1 = global_to_local[nncpair.first];
+                const int c2 = global_to_local[nncpair.second];
+                if (c1 < 0 || c2 < 0) {
```

Comment:
I do not think that specifiying NNCs with inactive cells should be fatal. But I could be wrong